### DISCUSSION TITLE
Feedback from Andrea Bizzotto

### DISCUSSION THREAD

**Discussion by RobertBrunhage** [RobertBrunhage/website#53] [General]
## 👋 Welcome!

This discussion will revolve around some feedback that we have gotten for the website from @bizz84. The discussion should be about how they can be resolved and what is the **most important**. The end goal here is to be able to make them in to issues so we have something to work from!

As promised here's some feedback on your site:

-  I like the little animation on the browse button, though I was expecting it to take me to the Featured video itself, not the video list. Maybe you could have separate "View" / "Browse all" buttons

- The section with Twitter references uses images which look blurred on retina display and are not accessibility-friendly. Should be easy to embed the actual tweets if that's what you're after.

- Course signup section is OK, but doesn't scream "I need to be on this list". You could borrow some copywriting ideas from [https://marketingexamples.com](https://marketingexamples.com) for this

- The videos page is simple but nice I like the hover effect. Though resizing could be better (when a layout breakpoint is crossed, the layout has a lot of empty space on the right)

- As a personal preference, I would add the published date on the video cards as well (not only on the content pages)

**Comment by RobertBrunhage** [comment]
> I like the little animation on the browse button, though I was expecting it to take me to the Featured video itself, not the video list. Maybe you could have separate "View" / "Browse all" buttons

Upvote this comment if you think we should have a "View" and "Browse all" button in the homepage.

**Comment by RobertBrunhage** [comment]
> The section with Twitter references uses images which look blurred on retina display and are not accessibility-friendly. Should be easy to embed the actual tweets if that's what you're after.

Do you have any good ideas regarding this one? These comments come from YouTube but maybe it's better to make custom components that should display this, what do you think?

Upvote for custom components

**Comment by RobertBrunhage** [comment]
> Course signup section is OK, but doesn't scream "I need to be on this list". You could borrow some copywriting ideas from https://marketingexamples.com for this

I am super bad at this so any help here is greatly appreciated! Here is how it looks currently

![image](https://user-images.githubusercontent.com/26344867/113252501-37a90080-92c4-11eb-8676-f820c5889216.png)

**Reply by sjmamani** [reply]
Maybe the message "When you sign up you are..." should be a title o subtitle, I mean make it more striking 🤔

**Reply by happyharis** [reply]
There must be an incentive to why I should give away my email. Incentives can be:
1. Free course/ebook/pdf
2. Early testers to courses

I believe the bigger problem is that you are not specific enough:
What kind of courses?

**Reply by RobertBrunhage** [reply]
True I agree, as of now I don't have an incentive as the course it just in a planning/designing phase.

But I guess I can try defining what that course is and add that instead!

**Reply by fadhilsaheer** [reply]
There should be a detailed explanation for everything, such as:

1.  why are you making this course 🤷‍♂️
2.  what is their benefits 🍎
3.  what does the course includes and  why is it special 🌟
etc ..

It will be nice if there is a specific page explaining this 📚

**Reply by RobertBrunhage** [reply]
> There should be a detailed explanation for everything, such as:
> 
> why are you making this course 🤷‍♂️
> what is their benefits 🍎
> what does the course includes and why is it special 🌟
> etc ..
> It will be nice if there is a specific page explaining this 📚

This will more be a thing when the course is out. Haven't had the time working on it so it's not set in stone yet :)

**Reply by RobertBrunhage** [reply]
Have now updated it a bit with a free PDF by signing up!

Thanks for the tip about PDF @happyharis

**Comment by RobertBrunhage** [comment]
> The videos page is simple but nice I like the hover effect. Though resizing could be better (when a layout breakpoint is crossed, the layout has a lot of empty space on the right)

I personally love that it's not a resizing animation but the centering could be resolved for certain breakpoints, what do you think?

**Comment by RobertBrunhage** [comment]
> As a personal preference, I would add the published date on the video cards as well (not only on the content pages)

Not much to add here more than, upvote or downvote if this is something that matters for you!
